src/components/VContentReport/VContentReportForm.vue

```diff
@@ -166,11 +167,13 @@ export default defineComponent({
     const isSubmitDisabled = computed(
       () => selectedReason.value === OTHER && description.value.length < 20
     )
-    const handleSubmit = async () => {
+    const handleSubmit = async (event) => {
+      event.preventDefault()
```

Comment:
```suggestion
      event.preventDefault()
```

Prevents redirection to the api POST request.